content/changelog/2022-10-04-console-release/index.md

```diff
@@ -0,0 +1,15 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- UI: Added social authentication for Hasura, enabling users to sign in using using social networking services supported by Hasura.
+- Improved Neon project creation performance.
+
+### Bug fixes
+
+- UI: Database selector allows to select any of created databases.
```

Comment:
```suggestion
- UI: The Database drop-down menu that appears in the Connection Details widget on the Dashboard and in the Neon SQL Editor now permits selecting any created database.
```

Reply:
Important: Please note that the file structure for the release notes changed just after you opened this PR. The contents of the changelog file must be moved to a new file at this path:
https://github.com/neondatabase/website/tree/main/content/release-notes/22-10-04.md